cc/libc/memset.cc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stddef.h>
+
+extern "C" {
+void *memset_avx2(void *str, int c, size_t n);
```

Comment:
Where is this actually defined?

Reply:
third_party/bionic/libc/arch-x86_64/string/avx2-memset-kbl.S

This file is built with multiple optimal versions of memset impl. and controls which one to use.